css/helpers/flexbox.css

```diff
@@ -17,7 +17,18 @@
   align-items: center;
 }
 
+
+/* Justify */
+.justify-sb{
+  justify-content: space-between;
+}
+
+
 /* flex gaps */
+.gap-05 {
+  gap: var(--size-000);
```

Comment:
using 0 like this looks confusing
if you are referring to miniscule values, use names instead

Reply:
The answer is given above, I may change this naming paradigm in the future or add a declaration about this in docs. Right now it makes me remember the variables easily.